src/components/Draggable/types/QuantityEnum.ts

```diff
@@ -0,0 +1,5 @@
+export enum QuantityEnum {
```

Comment:
I think those are the first enums created in the project, so either use the `Enum` sufix in both names or use singular names to create a pattern :blush:

Reply:
Ok !